src/vsr/replica.zig

```diff
@@ -784,7 +783,8 @@ pub fn ReplicaType(
 
             if (self.solo()) {
                 if (self.commit_min < self.op) {
-                    self.commit_journal(self.op);
+                    self.advance_commit_max(self.op, @src().fn_name);
```

Comment:
not sure whether I like `@src().fn_name`. On the one hand, it is cryptic and too clever. On the other hand, we did have bugs where we moved code around but didn't update call-sites. 

If we double-down here, we might pass the whole `@src()` in even